--- stm32cubeclt_setup_patch/../temp/install_clt_as_root.sh	2024-03-15 15:24:03.000000000 +0000
+++ stm32cubeclt_setup_patch/../temp/install_clt_as_root__patched__.sh	2024-04-04 19:31:48.607115807 +0000
@@ -16,6 +16,10 @@
 	uninstaller=$2
 	shift 2
 	(
+	if [[ $installer == st-stlink-server.*.install.sh ]]; then
+		echo "Overriding stlink-server install preferences with -f"
+		sh $installer -f
+	fi
 	sh $installer
 	case $? in
 	0)
